### PR TITLE
Support JIT configuration

### PR DESCRIPTION
**Pull request by mdboom**

### CODE REVIEW

File: bench_runner/result.py [hunk 1]
```diff
@@ -523,6 +523,21 @@ def github_action_url(self) -> Optional[str]:
...
+    @property
+    def hash_and_flags(self) -> str:
+        # A representation for the user that combines the commit hash and other flags
+        return " ".join(
+            [
+                self.cpython_hash,
+                *(self.is_tier2 and ["2️⃣"] or []),
+                *(self.is_jit and ["*️⃣"] or []),
+            ]
+        )
```
Comment (brandtbucher) [inline bench_runner/result.py:539]: I find this a bit easier to follow:

```suggestion
        parts = [self.cpython_hash]
        if self.is_tier2:
            parts.append("2️⃣")
        if self.is_jit:
            parts.append("*️⃣")
        return " ".join(parts)
```

File: bench_runner/scripts/should_run.py [hunk 2]
```diff
@@ -49,6 +50,8 @@ def main(
     flags = []
     if tier2:
         flags.extend(["PYTHON_UOPS"])
+    if jit:
+        flags.extend(["JIT"])
```
Comment (brandtbucher) [inline bench_runner/scripts/should_run.py:54]: You can just append here:

```suggestion
    if tier2:
        flags.append("PYTHON_UOPS")
    if jit:
        flags.append("JIT")
```

File: tests/test_get_merge_base.py [hunk 3]
```diff
@@ -39,7 +39,9 @@ def test_get_merge_base(tmp_path, capsys, checkout, monkeypatch):
 
     shutil.copytree(checkout / "cpython", tmp_path / "cpython")
 
-    get_merge_base.main(True, "linux-x86_64-linux", False, [], tmp_path / "cpython")
+    get_merge_base.main(
+        True, "linux-x86_64-linux", False, False, False, tmp_path / "cpython"
```
Comment (brandtbucher) [inline tests/test_get_merge_base.py:43]: Doesn't need to happen here, but I think the positional arguments are getting unwieldy enough that keyword-arguments could help here (and in the rest of the diff, below):

**Review by brandtbucher — APPROVED** [review]
Haven't had a chance to test it, but it looks good! Thanks. :)

A couple minor nits:

File: tests/test_get_merge_base.py (hunk 3 above)
Comment (mdboom) [reply tests/test_get_merge_base.py:43]: Yeah, this is kind of a weird one.  This is just a reflection of the positional arguments passed on the commandline from the Github Actions side (where positional arguments are a lot easier to deal with).

**Pull request merged**